drivers/storage/portworx/testspec/pks.yaml

```diff
@@ -51,7 +51,7 @@ spec:
             - name: "PX_TEMPLATE_VERSION"
               value: "v4"
             - name: "PRE-EXEC"
-              value: "if [ ! -x /bin/systemctl ]; then apt-get update; apt-get install -y systemd; fi"
+              value: "rm -fr /var/lib/osd/driver"
```

Comment:
So, we don't need the old PRE-EXEC command?

Reply:
That's correct -- it was added initially to support Ubuntu 14 PKS stemcells, which did not have the `systemd` package installed.  These are "long gone" now -- standard support for Ubuntu 14 LTS version ended beginning of 2019.